hazelcast/proxy/map.py

```diff
@@ -1283,6 +1284,28 @@ def remove(self, key: KeyType) -> Future[typing.Optional[ValueType]]:
 
         return self._remove_internal(key_data)
 
+    def remove_all(self, predicate: Predicate) -> Future[None]:
```

Comment:
From the Javadocs:
```
     * Note that calling this method also removes all entries from
     * caller's Near Cache.
```

So, I believe we should have some code changes in the near cached map as well